src/components/ValueTypeDropDown.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<select class="querybuilder__dropdown">
+		<option
+			class="querybuilder__dropdown-option"
+			v-for="(optionItem, index) in optionItems"
+			:value="optionItem.value"
+			:key="index"
+			@click="$emit( 'input', optionItem.value )"
+		>
+			{{ optionItem.value }}
+		</option>
+	</select>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend( {
+	name: 'ValueTypeDropDown',
+	data() {
+		return {
+			selected: null,
+			optionItems: [
+				{ value: 'matching' },
```

Comment:
When we switch to using the Dropdown, we probably will want to replace those with constants from an enum. But it is fine for now